samples/dependency-injection/externally-managed-mode/Core_10/Sample/Sample.csproj

```
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net10.0</TargetFramework>
+    <OutputType>Exe</OutputType>
+    <LangVersion>preview</LangVersion>
+    <EnablePreviewFeatures>true</EnablePreviewFeatures>
```

```suggestion
    <EnablePreviewFeatures>true</EnablePreviewFeatures>
```

Should we enable this?